### PR TITLE
Move dom stuff

### PR DESCRIPTION
**Pull request by 1602**
Addresses https://ubio-automation.atlassian.net/jira/software/projects/SDK/boards/16?selectedIssue=SDK-97 partially.

### CODE REVIEW

**Comment by qubyte** [comment]
With a rebase, is this still good to merge?

File: application-template/public/index.html [hunk 1]
```diff
@@ -25,7 +25,8 @@
 
                     form.remove();
 
-                    continueJob({ token, jobId, serviceId, input, local });
+                    const mountPoint = document.querySelector('#app');
```
[the remark this answers is not in the document]
Comment (1602) [reply application-template/public/index.html]: The idea here is to keep concerns separate, mountpoint is an argument of type HTMLNode, so we don't have to rely on same id name in across dependent pieces of code, and whether this is id or classname or something else is a separate concern, which should be taken care of in layout only.

File: application-template/public/index.js [hunk 2]
```diff
@@ -18,18 +20,64 @@ export async function createJob(/* will probably take arguments later */) {
     return res.json();
 }
 
-export async function continueJob({ token, jobId, serviceId, input, local }) {
+let tdsTimeout;
+
+function trackJobEvents(sdk) {
+    sdk.trackJob(async eventName/*, jobEvent */ => {
```
[the remark this answers is not in the document]
Comment (1602) [reply application-template/public/index.js]: Good catch

**Comment by 1602** [comment]
> A couple of comments. Mostly the duplication gives me pause. It means multiple places to potentially apply bug fixes. I can't think of a way out other than to revert to using shared (external still) templates.

This duplication bothers me too, but as far as I understood this was a conscious architecture decision we will not be changing as part of this particular task.

**Pull request merged**